src/app/helptext/storage/volumes/volume-status.ts

```diff
@@ -0,0 +1,73 @@
+import { T } from '../../../translate-marker';
+
+export default {
+    dialogFormFields: {
+        disk: {
+            placeholder: T('Member disk'),
+            tooltip: T(''),
+        },
+        passphrase: {
+            placeholder: T('Passphrase'),
+            tooltip: T('')
+        },
+        passphrase2: {
+            placeholder: T('Confirm Passphrase'),
+            tooltip: T(''),
+        },
+        force: {
+            placeholder: T('Force'),
+            tooltip: T(''),
+        },
+        new_disk: {
+            placeholder: T('New Disk'),
+            tooltip: T(''),
```

Comment:
Here's a rough suggestion - additional context could change what the message needs to say:

Select an unused disk to add to this vdev. WARNING: any data stored on the unused disk will be erased!